{"name": "test_create_order_invalid_ingredient", "status": "failed", "statusDetails": {"message": "AssertionError: assert 400 == 500\n +  where 400 = <Response [400]>.status_code", "trace": "self = <tests.test_order_creation.TestOrderCreation object at 0x000002B47570FD40>\nauthorized_user = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY2YTIyMmM5OWVkMjgwMDAxYjQ3YjM3MiIsImlhdCI6MTcyMTkwMTc2OSwiZXhwIjoxNzIxOTAyOTY5fQ.4IgD8Wl6Xp3m6m7gDG5E07azSZcaqTwpqLt1Qr__nFs'\n\n    @allure.story('Create order with invalid ingredient hash')\n    def test_create_order_invalid_ingredient(self, authorized_user):\n        api = StellarBurgersApi()\n        ingredients = [\"invalid_hash\"]\n    \n        response = api.create_order(ingredients, authorized_user)\n    \n>       assert response.status_code == 500\nE       assert 400 == 500\nE        +  where 400 = <Response [400]>.status_code\n\ntests\\test_order_creation.py:61: AssertionError"}, "start": 1721901769589, "stop": 1721901769730, "uuid": "8c34fe90-8ed0-48a7-9148-6c05d6560637", "historyId": "e4c7b16913966a6ce632447a4abfca8e", "testCaseId": "e4c7b16913966a6ce632447a4abfca8e", "fullName": "tests.test_order_creation.TestOrderCreation#test_create_order_invalid_ingredient", "labels": [{"name": "story", "value": "Create order with invalid ingredient hash"}, {"name": "feature", "value": "Order Creation"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_creation"}, {"name": "subSuite", "value": "TestOrderCreation"}, {"name": "host", "value": "DESKTOP-HPT8HPV"}, {"name": "thread", "value": "18704-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_creation"}]}